docs/release.md

```diff
@@ -30,20 +30,22 @@ git remote add upstream https://github.com/openshift/backplane-cli.git
 
 ### Cutting a new release
 
-Create a tag on the latest master.
+Create a tag on the latest main.
 
 ```bash
 git fetch upstream
-git checkout upstream/master
+git checkout upstream/main
 git tag -a ${VERSION} -m "release ${VERSION}"
 git push upstream $VERSION
```

Comment:
I had to use ssh to push to upstream.

Reply:
It worked for me without having to use SSH.
Perhaps something was off with the `GITHUB_TOKEN`?